interp/src/main.rs

```diff
@@ -4,6 +4,7 @@ use calyx::{
     pass_manager::PassManager,
     utils::OutputFile,
 };
+
```

Comment:
Is this space needed?

Reply:
Nope, think it got introduced during a merge conflict